.github/workflows/serverless-integration.yml

```diff
@@ -0,0 +1,39 @@
+name: "Serverless Integration Tests"
+
+on:
+  push:
+    paths:
+      - 'cmd/serverless/**'
+      - 'pkg/serverless/**'
+      - 'test/integration/serverless/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@v2
+        with:
+          path: go/src/github.com/DataDog/datadog-agent
+
+      - name: Set up Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      
+      - name: Install Serverless Framework
+        run: sudo yarn global add serverless --prefix /usr/local
+
+      - name: Checkout the datadog-lambda-extension repository
+        uses: actions/checkout@v2
+        with:
+          repository: DataDog/datadog-lambda-extension
+          path: go/src/github.com/DataDog/datadog-lambda-extension
+
+      - name: Run tests
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} 
```

Comment:
Secrets will not be accessible to Dependabot or external contributors. We should skip the tests when secrets are not available, to avoid broken CI

Reply:
I was not able to skip the tests in the workflow definition, because the `secrets` context is only available in very limited places (like setting environment variables) and not in an `if` condition. Instead, I updated the test script to `exit 0` when credentials are not found.

Reply:
Works for me!